modules/gitops-release-notes-1-8-5.adoc

```diff
@@ -0,0 +1,30 @@
+/ Module included in the following assembly:
```

Comment:
```suggestion
// Module included in the following assembly:
```
Missing a slash here. Content is showing up in the rendered preview.

Reply:
Thank you for the insight. Done